{{cookiecutter.app_name}}/README.md

```diff
@@ -12,12 +12,21 @@
 
 ### Development
 
-- Start development server
+#### Build dependencies
+This project using [`air`](https://github.com/cosmtrek/air) for live reloading. It need to be built as a binary file in `$GOPATH`.
+
+```sh
+make build-dependencies
+```
+
+#### Start development server
 
 ```sh
 make dev
 ```
 
+To visit app locally: `localhost:8080`
```

Comment:
Recheck the English again:

```suggestion
The application runs locally at http://localhost:8080
```

Reply:
Updated README in e8fbe849abed486a3a96b1ae6dd1f5c98959a54c